dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/LinkValidator.java

```diff
@@ -69,7 +69,7 @@ public void validate(ValidationHelper helper, Context context, String key, Link
                         if (operationRef.length != 4) {
                             isValid = false;
                         } else {
-                            String pathKey = operationRef[2].replace("~1", "/").replace("~0", "~");
+                            String pathKey = "/" + operationRef[2].replace("~1", "/").replace("~0", "~");
```

Comment:
All slashes in the path should be escaped with ~1. A slash shouldn't be added in-front. The test needs to be corrected to have ~1 (see comments in LinkValidatorTest)

Reply:
Also noticed that the checks above wouldn't catch invalid cases such as `#/invalid/~1myapp~1repositories/get`

So add a check like this:

change this:
```
String[] operationRef = t.getOperationRef().split("/");
if (operationRef.length != 4) {
   isValid = false;
```
to:

```
String[] operationRef;
if (!t.getOperationRef().startsWith("#/paths/") || (operationRef = t.getOperationRef().split("/")).length != 4) {
    isValid = false;
```